ipn/ipnlocal/local.go

```diff
@@ -2166,6 +2166,10 @@ func (b *LocalBackend) SetVarRoot(dir string) {
 // It returns an empty string if there's no configured or discovered
 // location.
 func (b *LocalBackend) TailscaleVarRoot() string {
+	if strings.ContainsRune(b.varRoot, ':') {
+		// Not writable.
```

Comment:
Worth adding an example of what is being detected here.

Reply:
Removed this, added tests for the generic prefix based lookup mechanism.